### PR TITLE
Updated to netstandard 2.0, updated nuget packages

### PR DESCRIPTION
**Pull request by rmcfarlane82**
this enables me to compile and use the library with azure functions

### CODE REVIEW

**Comment by stoiveyp** [comment]
As this would stop any future updates from being honoured in 1.x projects, can I ask what the problem is currently with Azure Functions?

I've got several .NET Core 2.0 projects using the existing NuGet builds without trouble so I'm guessing it's an infrastructure problem?

File: Alexa.NET.Tests/IntentTests.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public void Complex_intent_single_property()
 
 			Assert.Equal("AMAZON", name.Namespace);
 			Assert.Equal("SearchAction", name.Action);
-            Assert.Equal(1,name.Properties.Count);
+            Assert.True(name.Properties.Count == 1);
```
Comment (stoiveyp) [inline Alexa.NET.Tests/IntentTests.cs]: It's good to be tidying up the assertions, could we remove the logic and use Assert.Single instead?

**Comment by timheuer** [comment]
@rmcfarlane82 the existing build shouldn't prevent you from using in existing Azure Functions.  In fact I had checked this a while back when we discovered Functions didn't support NET Standard at all.  But they do now and this being NS 1.6 is compatible with Azure Functions 2.0 platform which supports NET Standard.

**Comment by rmcfarlane82** [comment]
@timheuer @stoiveyp for some reason on our old machines with a few versions of visual studio installed our solutions compile but on fresh machines with a fresh copy of visual studio and the up to date azure functions SDK our solutions stop building, as soon as i update this project to standard the solutions compile. we were getting System.Runtime not found and Newtonsoft.Json incompatible errors. any suggestions are welcome

File: Alexa.NET.Tests/IntentTests.cs (hunk 1 above)
Comment (rmcfarlane82) [reply Alexa.NET.Tests/IntentTests.cs]: Assert.Single would be the best solution

**Comment by rmcfarlane82** [comment]
@stoiveyp yeah sure

Severity	Code	Description	Project	File	Line	Suppression State
Error		System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
File name: 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at System.Signature.GetSignature(Void* pCorSig, Int32 cCorSig, RuntimeFieldHandleInternal fieldHandle, IRuntimeMethodInfo methodHandle, RuntimeType declaringType)
   at System.Reflection.RuntimeMethodInfo.FetchNonReturnParameters()
   at System.Reflection.RuntimeMethodInfo.GetParameters()
   at MakeFunctionJson.MethodInfoExtensions.GetDisabled(MethodInfo method)
   at MakeFunctionJson.MethodInfoExtensions.HasUnsuportedAttributes(MethodInfo method, String& error)
   at MakeFunctionJson.FunctionJsonConverter.TryGenerateFunctionJsons()
   at MakeFunctionJson.FunctionJsonConverter.TryRun()

Error generating functions metadata

**Comment by stoiveyp** [comment]
hmm - I can't get it to fail on any of my installations, but there are threads out there with projects having similar issues (.NETStandard 2.0 referencing 1.x and having similar issues)

Only solution I've been able to find that works in other scenarios is adding the following to the csproj file
```
<PropertyGroup>
    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
</PropertyGroup>
```

**Comment by rmcfarlane82** [comment]
@stoiveyp Thanks i'll give it a go in a few hours and let you know how it went.

**Pull request closed without merging**